lib/src/models/multireddit.dart

```diff
@@ -3,13 +3,398 @@
+import 'package:draw/src/user.dart';
+
+enum Visibility { hidden, private, public }
+
+String _visibilityToString(Visibility visibility) {
+  switch (visibility) {
+    case Visibility.hidden:
+      return "hidden";
+    case Visibility.private:
+      return "private";
+    case Visibility.public:
+      return "public";
+    default:
+      throw new DRAWInternalError('Visiblitity: $visibility is not supported');
+  }
+}
+
+enum WeightingScheme { classic, fresh }
+
+String _weightingSchemeToString(WeightingScheme weightingScheme) {
+  switch (weightingScheme) {
+    case WeightingScheme.classic:
+      return "classic";
+    case WeightingScheme.fresh:
+      return "fresh";
+    default:
+      throw new DRAWInternalError(
+          'WeightingScheme: $weightingScheme is not supported');
+  }
+}
+
+enum IconName {
+  artAndDesign,
+  ask,
+  books,
+  business,
+  cars,
+  comic,
+  cuteAnimals,
+  diy,
+  entertainment,
+  foodAndDrink,
+  funny,
+  games,
+  grooming,
+  health,
+  lifeAdvice,
+  military,
+  modelsPinup,
+  music,
+  news,
+  philosophy,
+  picturesAndGifs,
+  science,
+  shopping,
+  sports,
+  style,
+  tech,
+  travel,
+  unusualStories,
+  video,
+  emptyString,
+  none,
+}
+
+String _iconNameToString(IconName iconName) {
+  switch (iconName) {
+    case IconName.artAndDesign:
+      return "art and design";
+    case IconName.ask:
+      return "ask";
+    case IconName.books:
+      return "books";
+    case IconName.business:
+      return "business";
+    case IconName.cars:
+      return "cars";
+    case IconName.comic:
+      return "comics";
+    case IconName.cuteAnimals:
+      return "cute animals";
+    case IconName.diy:
+      return "diy";
+    case IconName.entertainment:
+      return "entertainment";
+    case IconName.foodAndDrink:
+      return "food and drink";
+    case IconName.funny:
+      return "funny";
+    case IconName.games:
+      return "games";
+    case IconName.grooming:
+      return "grooming";
+    case IconName.health:
+      return "health";
+    case IconName.lifeAdvice:
+      return "life advice";
+    case IconName.military:
+      return "military";
+    case IconName.modelsPinup:
+      return "models pinup";
+    case IconName.music:
+      return "music";
+    case IconName.news:
+      return "news";
+    case IconName.philosophy:
+      return "philosophy";
+    case IconName.picturesAndGifs:
+      return "pictures and gifs";
+    case IconName.science:
+      return "science";
+    case IconName.shopping:
+      return "shopping";
+    case IconName.sports:
+      return "sports";
+    case IconName.style:
+      return "style";
+    case IconName.tech:
+      return "tech";
+    case IconName.travel:
+      return "travel";
+    case IconName.unusualStories:
+      return "unusual stories";
+    case IconName.video:
+      return "video";
+    case IconName.emptyString:
+      return "";
+    case IconName.none:
+      return "None";
```

Comment:
Is this uppercase intentionally?

Reply:
Yup 😄 

Reference: https://www.reddit.com/dev/api/#DELETE_api_filter_{filterpath}